api/src/main/resources/liquibase.xml

```diff
@@ -217,4 +217,11 @@
 			UPDATE global_property SET property_value = 'false' WHERE property IN ( 'ugandaemrfingerprint.started', 'fhir.started', 'dataentrystatistics.started');
 		</sql>
 	</changeSet>
+
+	<changeSet id="ugandaemr-01042019-1305" author="dbaluku">
```

Comment:
@dbaluku Just remove the ugandaemr.fingerprint.started from the previous query - u do not need a new changeset

Also change the ID of the changeset otherwise it will have checksum issues

Reply:
@ssmusoke @dbaluku I suggest a check of the gp existence before execution of the liquibase. Since its changing  an existing value

Reply:
Removed the ugandaemr.fingerprint.started from the previous query